src/app/collections/PlugSet.tsx

```diff
@@ -24,64 +24,64 @@ interface Props {
 /**
  * A single plug set.
  */
-export default class PlugSet extends React.Component<Props> {
-  render() {
-    const { defs, buckets, plugSetHash, items, path, onNodePathSelected } = this.props;
+export default function PlugSet({
+  defs,
+  buckets,
+  plugSetHash,
+  items,
+  path,
+  onNodePathSelected
+}: Props) {
+  const plugSetDef = defs.PlugSet.get(plugSetHash);
 
-    const plugSetDef = defs.PlugSet.get(plugSetHash);
+  const vendorItems = plugSetDef.reusablePlugItems.map((i) =>
+    VendorItem.forPlugSetItem(
+      defs,
+      buckets,
+      i,
+      items.some((k) => k.plugItemHash === i.plugItemHash && k.enabled)
+    )
+  );
 
-    const vendorItems = plugSetDef.reusablePlugItems.map((i) =>
-      VendorItem.forPlugSetItem(
-        defs,
-        buckets,
-        i,
-        items.some((k) => k.plugItemHash === i.plugItemHash && k.enabled)
-      )
-    );
+  const acquired = count(vendorItems, (i) => i.canPurchase);
+  const childrenExpanded = path.includes(plugSetHash);
 
-    const acquired = count(vendorItems, (i) => i.canPurchase);
-    const childrenExpanded = path.includes(plugSetHash);
+  const title = (
+    <span className="node-name">
+      <BungieImage src={defs.InventoryItem.get(3960522253).displayProperties.icon} />{' '}
```

Comment:
should we set this as EMOTE_PLUG_HASH, not that that has anything to do with this PR just noticed it tho

Reply:
definitely